tests/utils.go

```diff
@@ -39,7 +39,7 @@ import (
 	"time"
 
 	ghodssyaml "github.com/ghodss/yaml"
-	"github.com/google/goexpect"
+	expect "github.com/google/goexpect"
```

Comment:
use explicit names for packages only when you have a conflict with another package

Reply:
VSCode entered these explicit package names automatically once I saved each of these files, so I couldn't even remove them (unless I stopped using vscode).

Reply:
Resolved this issue in VSCode with the help of Meni.